packages/contracts/src/utils/JwtVerifier.sol

```diff
@@ -89,6 +91,28 @@ contract JwtVerifier is IVerifier, OwnableUpgradeable, UUPSUpgradeable {
             ? 1
             : 0;
 
+        // Check JwtRegistry
+        if (
+            jwtRegistry.isJwtPublicKeyValid(
+                proof.domainName,
+                proof.publicKeyHash
+            )
+        ) {
+            jwtRegistry.updateJwtRegistry();
+        }
```

Comment:
@wshino If `jwtRegistry.isJwtPublicKeyValid` returns true, we should not update the registry, we should only update it if the function returns `false`.

Reply:
```suggestion
        if (
            !jwtRegistry.isJwtPublicKeyValid(
                proof.domainName,
                proof.publicKeyHash
            )
        ) {
            jwtRegistry.updateJwtRegistry();
        }
```

Reply:
@shreyas-londhe Good catch. Thank you.